src/queue.c

```diff
@@ -154,7 +154,10 @@ struct dispatch_root_queue_context_s {
 			int volatile dgq_pending;
 #if DISPATCH_USE_WORKQUEUES
 			qos_class_t dgq_qos;
-			int dgq_wq_priority, dgq_wq_options;
+#if HAVE_PTHREAD_WORKQUEUES
```

Comment:
I think this could be conditionalized more tightly with `HAVE_PTHREAD_WORKQUEUE_SETDISPATCH_NP || DISPATCH_USE_LEGACY_WORKQUEUE_FALLBACK` now (or add a define at the top for this combination to make it cleaner), those are the only remaining cases that need this field)